packages/template-retail-react-app/app/components/list-menu/index.jsx

```diff
@@ -217,40 +220,48 @@ ListMenuPopover.propTypes = {
  * The submenus are open when the user moves the mouse over the trigger and for A11y when
  * users use the keyboard Tab key to focus over the chevron icon and press Enter.
  *
- * @param root The root category in your commerce cloud back-end.
  * @param maxColumns The maximum number of columns that we want to use per row inside the ListMenu.
  */
-const ListMenu = ({root, maxColumns = MAXIMUM_NUMBER_COLUMNS}) => {
+const ListMenu = ({maxColumns = MAXIMUM_NUMBER_COLUMNS}) => {
+    const {root, itemsKey} = useCategories()
     const theme = useTheme()
     const {baseStyle} = theme.components.ListMenu
+    const [ariaBusy, setAriaBusy] = useState('true')
 
-    const itemsKey = 'categories'
-    const items = root ? root[itemsKey] : false
+    useEffect(() => {
+        setAriaBusy('false')
+    }, [])
```

Comment:
I might have mislead you in one of my previous comments, but it seems weird to have `ariaBusy` default to `true`, and then immediately set it to false after mount in this useEffect? 

I'd imagine the busy state should based on if the fetch category promise is resolved.

Reply:
I don't think we need `useState` at all for this. We just need

```javascript
       <nav
            id="list-menu"
            aria-label="main"
            aria-live="polite"
            aria-busy={root?.[itemsKey]?.filter(item => !item.loaded)?.length > 0}
            aria-atomic="true"
        >
```